examples/select-table-row-by-cell-content/README.md

```diff
@@ -0,0 +1,27 @@
+# Select a Table Row by Cell Content
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+This example shows how to select a table row based on the cell content.
+
+The tested page includes a table with `Id`, `First name` and `Last name` columns and three rows of data. A click on a row highlights it.
+
+The test identifies the target row by the value of its `Id` column. Then it uses the `t.eql` method to assert that the row has expected cell values and the `t.ok` method to check that it is highlighted.
```

Comment:
> to assert that the row has expected cell values

1) I don't think *assert that* is wrong here, but the correctors will change it to *verify that*.
2) There will also be an issue with *expected cell values*. It may seem natural to introduce the word *expected* like that for us who write about testing, but the correctors are going to ask who and why expects these values. 

With this in mind, I'd simply say: *Then it uses the `t.eql` method to verify cell values in this row and...*

Reply:
edited